### PR TITLE
cleaned up preferences

### PR DESCRIPTION
**Pull request by weissercn**

### CODE REVIEW

**Comment by weissercn** [comment]
Need to still run some tests

File: influence_benchmark/stats/preferences_per_iteration.py [hunk 1]
```diff
@@ -76,9 +76,9 @@ def calculate_expectation(score_distribution: Dict[str, float]) -> float:
     return sum(float(score) * probability for score, probability in score_distribution.items())
 
 
-def process_iteration_data(trajectory_path: Path, top_n: int) -> Optional[Tuple[int, float, float, float, float]]:
+def process_iteration_data(trajectory_path: Path, top_n: int) -> Dict[str, Union[Tuple[List[Dict]], int, float]]:
     """Process data for a single iteration.
-    Returns
+    Returns a dict containing
         top_n_trajs_df: data for the top n trajectories
```
Comment (micahcarroll) [inline influence_benchmark/stats/preferences_per_iteration.py:82]:
```suggestion
        top_n_trajs_dict: data for the top n trajectories
```

File: influence_benchmark/stats/preferences_per_iteration.py [hunk 2]
```diff
@@ -117,40 +125,42 @@ def analyze_run(
...
-            all_infl_avg_top_trajs.append(infl_avg_top_trajs)
+            for key in [
+                "top_n_trajs_dict",
+                "n_trajs",
+                "rew_avg_all_trajs",
+                "rew_avg_top_trajs",
+                "infl_avg_all_trajs",
+                "infl_avg_top_trajs",
+            ]:
+                trajs[key].append(result[key])
```
Comment (micahcarroll) [inline influence_benchmark/stats/preferences_per_iteration.py:145]:
```suggestion
            for k, v in result.items():
                trajs[k].append(v)
```

File: influence_benchmark/stats/preferences_per_iteration.py [hunk 3]
```diff
@@ -117,40 +125,42 @@ def analyze_run(
...
         valid_iterations,
-        all_rew_avg_all_trajs,
-        all_rew_avg_top_trajs,
-        all_infl_avg_all_trajs,
-        all_infl_avg_top_trajs,
+        trajs["rew_avg_all_trajs"],
+        trajs["rew_avg_top_trajs"],
+        trajs["infl_avg_all_trajs"],
+        trajs["infl_avg_top_trajs"],
     )
```
Comment (micahcarroll) [inline influence_benchmark/stats/preferences_per_iteration.py:166]: Not worth the time to change it rn, but maybe this could have also returned a dict?

**Review by micahcarroll — APPROVED** [review]

File: influence_benchmark/stats/preferences_per_iteration.py (hunk 2 above)
Comment (weissercn) [reply influence_benchmark/stats/preferences_per_iteration.py]: fixed

File: influence_benchmark/stats/preferences_per_iteration.py (hunk 3 above)
Comment (weissercn) [reply influence_benchmark/stats/preferences_per_iteration.py]: fixed

**Comment by weissercn** [comment]
@micahcarroll please review again after changes

**Pull request merged**